apps/mobile-alphabet-e2e/.babelrc

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Can you try to sort out the division of labour between Babel, Webpack, and Metro in this project? You know how nervous complex build chains make me.

Reply:
Not saying to do anything differently, just try to have a mental model of what's actually going on with the build.